modin/config/envvars.py

```diff
@@ -239,6 +239,16 @@ class TestDatasetSize(EnvironmentVariable, type=str):
     choices = ("Small", "Normal", "Big")
 
 
+class TestRayClient(EnvironmentVariable, type=bool):
```

Comment:
As a side note here, we should probably have configs specific for tests elsewhere (not needed to change here).